benching/mod.ts

```diff
@@ -14,10 +14,10 @@ export interface BenchmarkTimer {
 }
 
 /** Defines a benchmark through a named function. */
-export type BenchmarkFunction = {
+export interface BenchmarkFunction {
```

Comment:
oof wtf...

Reply:
The defaults for eslint for TypeScript specify that when something looks like an interface, it is preferred over a type alias.  I agree with that default.

Reply:
Oh, I didn't even know about the other way of specifying an interface. The "wtf" was meant more to myself like: "wtf I approved this?"